src/test/java/gov/usgs/owi/nldi/controllers/HtmlControllerIT.java

```diff
@@ -54,6 +54,128 @@ public void getLinkedDataHtmlTest() throws Exception {
 		assertTrue(actualbody.contains("a>"));
 	}
 
+	@Test
+	public void getLinkedDataWithQueryStringTest() throws Exception {
+		String actualbody = assertEntity(restTemplate,
+			"/linked-data/comid/position?coords=POINT(-89.35 48.064)&f=html",
+			HttpStatus.OK.value(),
+			null,
+			null,
+			null,
+			null,
+			false,
+			false);
+		assertTrue(actualbody.contains("/linked-data/comid/position?f=json&coords=POINT(-89.35%2048.064)"));
+		assertTrue(actualbody.contains("<html>"));
+		assertTrue(actualbody.contains("</html"));
+		assertTrue(actualbody.contains("<a "));
+		assertTrue(actualbody.contains("href="));
+		assertTrue(actualbody.contains("a>"));
+	}
+
+	@Test
+	public void getLinkedDataWithAlternateQueryStringTest() throws Exception {
+		String actualbody = assertEntity(restTemplate,
+			"/linked-data/comid/position?f=html&coords=POINT(-89.35 48.064)",
+			HttpStatus.OK.value(),
+			null,
+			null,
+			null,
+			null,
+			false,
+			false);
+		assertTrue(actualbody.contains("/linked-data/comid/position?f=json&coords=POINT(-89.35%2048.064)"));
+		assertFalse(actualbody.contains("f=html"));
+		assertTrue(actualbody.contains("<html>"));
+		assertTrue(actualbody.contains("</html"));
+		assertTrue(actualbody.contains("<a "));
+		assertTrue(actualbody.contains("href="));
+		assertTrue(actualbody.contains("a>"));
+	}
+
+	@Test
+	public void getLinkedDataDataSourcesTest() throws Exception {
+		String actualbody = assertEntity(restTemplate,
+			"/linked-data?f=html",
+			HttpStatus.OK.value(),
+			null,
+			null,
+			null,
+			null,
+			false,
+			false);
+		assertTrue(actualbody.contains("/linked-data?f=json"));
+		assertFalse(actualbody.contains("f=html"));
+		assertTrue(actualbody.contains("<html>"));
```

Comment:
is the document expected to start with "<html>"?  Or could comments header be added?
if so, the .startsWith() could be used instead.